packages/auth/src/AuthProvider.tsx

```diff
@@ -95,6 +95,7 @@ type AuthProviderProps =
       config?: SupportedAuthConfig
       skipFetchCurrentUser?: boolean
     }
+) & { children?: React.ReactNode }
```

Comment:
Just like we already have `skipFetchCurrentUser` on all the different types already I'd repeat `children` on them too.